FinniversKit/Sources/Components/BrazePromotionView/BrazePromotionView.swift

```diff
@@ -0,0 +1,249 @@
+import UIKit
+
+public protocol BrazePromotionViewDelegate: AnyObject {
+    func brazePromotionView(_ brazePromotionView: BrazePromotionView, didSelect action: BrazePromotionView.Action)
+    func brazePromotionViewTapped(_ brazePromotionView: BrazePromotionView)
+}
+
+public class BrazePromotionView: UIView {
+
+    private let buttonSize = 26.0
+    private let buttonTouchSurfaceSize = 44.0
```

Comment:
This property is unused and can be removed

Reply:
Yes! Thanks to the method you shared with me for increasing the touch surface of the button, I no longer need this 🎉